enterprise/functions/src/main/java/io/crate/window/FirstLastValueFunction.java

```diff
@@ -73,13 +81,28 @@ public Object execute(int rowIdx,
     }
 
     public static void register(EnterpriseFunctionsModule module) {
-        module.register(NAME, new BaseFunctionResolver(FuncParams.SINGLE_ANY) {
```

Comment:
Would it make sense to have an eNum or something and then iterate for the registration? Both functions have the same signature.